lib/text_cleanup/test/job_log.t/run.t

```diff
@@ -452,3 +452,12 @@
   user command error: exit status 1
   
   "
+  $ dune exec text_cleanup -- log3 | sed 's/\$/%/g' | sed 's/\\n/\n/g'
+  "
+  Installing to existing venv 'something'
+  
+  
+  [09:52:01 #] sudo aptitude install -y package1 package2
+  
+  \\027
```

Comment:
what is the use case for seeing this? why not ignore/remove it?

Reply:
Because there are a lot of escape code, and probably missed some. Escape codes start with \027 and then have some characters. The other chars could be something like `[25l`. printing `\\027[25l` make what happened obvious for future debuggers.